frontend/simple/js/events.js

```diff
@@ -3,13 +3,26 @@ import * as Events from '../../../shared/events'
 import store from './state'
 import backend from '../js/backend'
 import {namespace} from '../js/backend/hapi'
+import _ from 'lodash'
 
 export class GroupContract extends Events.HashableGroup {
   static vuex = GroupContract.Vuex({
-    state: { votes: [], payments: [], members: [], invitees: [], profiles: {} },
+    state: { proposals: {}, payments: [], members: [], invitees: [], profiles: {} },
     mutations: {
       HashableGroupPayment (state, {data}) { state.payments.push(data) },
-      HashableGroupVote (state, {data}) { state.votes.push(data) },
+      HashableGroupProposal (state, {data, hash}) { state.proposals[hash] = {...data, for: [], against: []} },
+      HashableGroupVoteForProposal (state, {data}) {
+        if (state.proposals[data.proposalHash]) {
+          state.proposals[data.proposalHash].for.push(data.username)
+          if (state.proposals[data.proposalHash].for.length >= state.proposals[data.proposalHash].threshold) { Vue.delete(state.proposals, data.proposalHash) }
```

Comment:
so the way voting is going to be done can vary depending on the voting method.

by default the threshold will probably be a percentage, above which the proposal is considered to have passed.

Can you:

- Modify this to be in terms of percentage for? e.g. a threshold of 50% passes when > 50% have voted in favor.
- Add a note/todo here to mention that other types of voting methods are possible. e.g. a proposal can pass if at least `n` number of people have voted for it (what you're currently doing here), or something else